doc/source/configure_globus_transfertool.rst

```diff
@@ -0,0 +1,103 @@
+Configure Rucio To Use Globus Online as a Transfer Tool
+=======================================================
+
+This document walks through an example configuration of Rucio to use Globus Online as a transfer tool. There are four configuration points shown here: registration of your application with Globus, RSE setup (properties and parameters), the Rucio configuration file `rucio.cfg` and the Globus configuration file `config.yml`.
+
+Use of both Globus Server endpoints and Globus Personal endpoints has been tested with the below approach.  Creation of the Globus endpoints is outside the scope here.  Some knowledge of Rucio setup and familiarity with Globus configuration is presumed.
+
+Register Application with Globus
+--------------------------------
+
+Using Globus Online as a transfer tool requires `registering <https://developers.globus.org>`_ the client application with Globus Online.  Be sure to select Native App and include a scope for `urn:globus:auth:scope:transfer.api.globus.org:all`.  Once you have the Client ID you’ll need to install the globus sdk and run the below Python code to obtain a refresh token.
+
+There is a `helpful walk-through <https://globus-sdk-python.readthedocs.io/en/stable/tutorial/>`_ that goes into more detail around OAuth and token retrieval.
+
+Obtain a refresh token to access Globus resources::
```

Comment:
For this part, where does it go effectively? Why does the user need to know this?

Reply:
Refresh token is stored and retrieved from ./lib/rucio/transfertool/config.yml (detailed below).  A user would not need to know this but an administrator setting up Rucio would need to know how to authenticate to GO.  Should this document in general be put somewhere else?

Reply:
That config.yml we should move to `./etc/` where the other Rucio configs are located. @maatthias can you submit a separate PR for this. I will merge this one in the meanwhile.